website/content/docs/deprecation/index.mdx

```diff
@@ -28,7 +28,7 @@ This announcement page is maintained and updated periodically to communicate imp
 | AAD Graph on Azure Secrets Engine | v1.10                    | 1.11           | v1.12           | Microsoft will end its support of the [AAD Graph API on June 30, 2022](https://docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-overview). Support for Microsoft Graph API was introduced in Vault 1.9. If your Vault deployment is on a prior release, you may use the Azure Secrets Engine as an external plugin while you plan to upgrade. | [AAD (Azure Active Directory](https://vault-git-post-1-10-doc-changes-hashicorp.vercel.app/docs/secrets/azure#aad-azure-active-directory)                                                                                                                        |
 | Optional `api_token` parameter in Okta Auth Method  | v1.4                    | 1.11           | v1.12           | The `api_token` parameter on the Okta Auth Method will change from being optional to being required. | [API Documentation](https://www.vaultproject.io/api-docs/auth/okta#api_token) |
-| SHA-1 certificate signing | v1.11             | v1.11          | v1.12           | Go version 1.18 removes support for SHA-1 by default. As Vault updates its Go version to 1.18, you should plan to move off SHA-1 for certficate signing. Operators can set a Go [environmental variable]((/docs/deprecation/faq#q-what-is-the-impact-of-removing-support-for-x-509-certificates-with-signatures-that-use-sha-1)) to restore SHA-1 support if they need to continue using SHA-1. It is unknown at this time when Go will remove the environmental variable support (current projection is removal in Go 1.19). Therefore, we highly encourage you to migrate off of SHA-1 for certificate signing before Vault adopts Go 1.19.|[FAQ](/docs/deprecation/faq#q-what-is-the-impact-of-removing-support-for-x-509-certificates-with-signatures-that-use-sha-1)|
+| SHA-1 certificate signing | v1.11             | v1.11          | v1.12           | Go version 1.18 removes support for SHA-1 by default. As Vault updates its Go version to 1.18, you should plan to move off SHA-1 for certficate signing. Operators can set a Go [environmental variable](/docs/deprecation/faq#q-what-is-the-impact-of-removing-support-for-x-509-certificates-with-signatures-that-use-sha-1) to restore SHA-1 support if they need to continue using SHA-1. It is unknown at this time when Go will remove the environmental variable support (current projection is removal in Go 1.19). Therefore, we highly encourage you to migrate off of SHA-1 for certificate signing before Vault adopts Go 1.19.|[FAQ](/docs/deprecation/faq#q-what-is-the-impact-of-removing-support-for-x-509-certificates-with-signatures-that-use-sha-1)|
```

Comment:
We're now going to use Go 1.19 for Vault 1.12, does this impact the text?

Reply:
Oh, nvm, I see that you didn't write this text, and it's not the subject of the PR.